crates/kitsune_p2p/direct/src/v1.rs

```diff
@@ -899,55 +897,56 @@ async fn handle_query_op_hashes(
     //        we'll want an api to just get the hashes
     let mut entries = vec![];
 
-    for (agent, arcset) in agents {
-        let agent = KdHash::from_kitsune_agent(&agent);
-        let es = kdirect
-            .persist
-            .query_entries(root.clone(), agent, window.clone(), arcset)
-            .await?;
-        entries.extend(es.into_iter());
-    }
-
-    let mut entries: Vec<_> = entries
-        .into_iter()
-        .map(|e| e.hash().clone().to_kitsune_op_hash())
-        .collect();
-    entries.sort();
-    entries.dedup();
+    // let agents = todo!();
+    // for (agent, arcset) in agents {
+    //     let agent = KdHash::from_kitsune_agent(&agent);
+    //     let es = kdirect
+    //         .persist
+    //         .query_entries(root.clone(), agent, window.clone(), arcset)
+    //         .await?;
+    //     entries.extend(es.into_iter());
+    // }
+
+    // let mut entries: Vec<_> = entries
+    //     .into_iter()
+    //     .map(|e| e.hash().clone().to_kitsune_op_hash())
+    //     .collect();
+    // entries.sort();
+    // entries.dedup();
 
     // TODO: produce proper time window of actual data returned
     Ok(Some((entries, window)))
 }
 
 async fn handle_fetch_op_data(
-    kdirect: Arc<Kd1>,
-    input: FetchOpDataEvt,
+    _kdirect: Arc<Kd1>,
+    _input: FetchOpDataEvt,
 ) -> KdResult<Vec<(Arc<KitsuneOpHash>, Vec<u8>)>> {
-    let FetchOpDataEvt {
-        space,
-        agents,
-        op_hashes,
-        ..
-    } = input;
-
-    let mut out = Vec::new();
-    let root = KdHash::from_kitsune_space(&space);
-
-    for op_hash in op_hashes {
-        for agent in agents.iter() {
-            let agent = KdHash::from_kitsune_agent(agent);
-            let hash = KdHash::from_kitsune_op_hash(&op_hash);
-            if let Ok(entry) = kdirect
-                .persist
-                .get_entry(root.clone(), agent.clone(), hash)
-                .await
-            {
-                out.push((op_hash.clone(), entry.as_wire_data_ref().to_vec()));
-            }
-        }
-    }
-
-    Ok(out)
+    // let FetchOpDataEvt {
```

Comment:
more commented code

Reply:
Fixed in https://github.com/holochain/holochain/commit/140576f89e08d81a09e9c21a2a84d9d17a66482b